templates/base.html

```diff
@@ -0,0 +1,98 @@
+{% load static %}
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+
+    <!-- Boostrap basic imports -->
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet"
```

Comment:
Why do you have two imports of bootstrap? I see it here and in <script> tag (line 81)

Reply:
According to:
[https://getbootstrap.com/docs/4.3/getting-started/introduction/](https://getbootstrap.com/docs/4.3/getting-started/introduction/)

...bootstrap.min.css - import for CSS (pasted in `<head>` tag)
...bootstrap.min.js - import for JS (pasted before the end of `<body>` tag)